app/src/main/java/ch/epfl/sweng/erpa/services/dummy/database/DummyGameService.java

```diff
@@ -0,0 +1,124 @@
+package ch.epfl.sweng.erpa.services.dummy.database;
+
+import android.content.Context;
+
+import com.annimon.stream.Optional;
+
+import org.yaml.snakeyaml.Yaml;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.FileReader;
+import java.io.OutputStreamWriter;
+import java.util.ArrayList;
+import java.util.List;
+
+import ch.epfl.sweng.erpa.model.Game;
+
+public class DummyGameService implements ch.epfl.sweng.erpa.services.GameService
+{
+
+    private final static String EXTENSION = ".yaml";
+    private final static String GAME_DATA_FOLDER = "game_data";
```

Comment:
Please rename to `SAVED_GAME_FILES_FOLDER` and the value should be something along the lines `saved_games`